<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="header">
      <property name="styleSheet">
       <string notr="true">background-color:qlineargradient(spread:pad, x1:0.341, y1:0.034, x2:1, y2:0, stop:0 rgba(153, 0, 0, 245), stop:1 rgba(122, 17, 17, 246))</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Trueno</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
color:#fff</string>
         </property>
         <property name="text">
          <string>O-LIBRARY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Trueno</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
color:#fff</string>
         </property>
         <property name="text">
          <string>'PINJAM-PINJAM BUKU JADI MUDAH'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="body">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(170, 255, 0)rgb(85, 0, 0)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QFrame" name="judul_4">
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 255, 127)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <family>Trueno</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:none;
color:rgb(170, 0, 0);
</string>
            </property>
            <property name="text">
             <string>CHANGE PASSWORD</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="input_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#fff</string>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStatesMinorOutlyingIslands"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>80</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Roboto Bk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Username :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>162</x>
            <y>237</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Roboto Bk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>E-mail :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>161</x>
            <y>160</y>
            <width>90</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Roboto Bk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Password :</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="usr_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>67</y>
            <width>351</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="psw_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>147</y>
            <width>350</width>
            <height>41</height>
           </rect>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="email_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>231</y>
            <width>350</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="vpsw_3">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>317</y>
            <width>311</width>
            <height>41</height>
           </rect>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>  Verify Password...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="go">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>317</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(162, 162, 162);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/arrow-right.svg</normaloff>:/icons/icons/arrow-right.svg</iconset>
          </property>
         </widget>
         <widget class="QLabel" name="error_regis_3">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>15</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QGroupBox" name="changemenu">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>781</width>
            <height>451</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="title">
           <string>change password</string>
          </property>
          <widget class="QLineEdit" name="ps">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>170</y>
             <width>281</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>  Your New Password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="con">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>230</y>
             <width>281</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>  Confirm Your Password...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="change">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>300</y>
             <width>141</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(111, 111, 111);</string>
           </property>
           <property name="text">
            <string>Change Password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="ps_2">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>110</y>
             <width>281</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>  Your Old Password</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="footer">
      <property name="styleSheet">
       <string notr="true">background-color:qlineargradient(spread:pad, x1:0.341, y1:0.034, x2:1, y2:0, stop:0 rgba(153, 0, 0, 245), stop:1 rgba(122, 17, 17, 246))</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
color:#fff</string>
         </property>
         <property name="text">
          <string>O-Library Patch 0.0 beta</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
